src/studio/src/designer/backend.Tests/Services/SchemaModelServiceTests.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Factories;
+using Altinn.Studio.Designer.Services.Implementation;
+using Altinn.Studio.Designer.Services.Interfaces;
+using Designer.Tests.Utils;
+using FluentAssertions;
+using Xunit;
+
+namespace Designer.Tests.Services
+{
+    public class SchemaModelServiceTests
+    {
+        [Fact]
+        public async Task DeleteSchema_AppRepo_ShouldDelete()
```

Comment:
This test failed for me during first run. "File already used by another process". We should be careful using file access in tests that can run in parallel.

Reply:
Hm, thanks. I'll look into it. Tests should run on a copy isolated from other tests, especially those that modify data.

Reply:
I see I do a copy in this case. That means it's the read operation that fail which is strange because we don't lock on read (or at least don't intend to). I'll have a closer look so we don't end up with fragile tests.

Reply:
I've tried to fix this now by creating a new method for reading the files allowing to specify the lock and share levels.

Reply:
👍